src/diffTool/diffProcessor.ts

```diff
@@ -49,17 +50,9 @@ export async function diffRaml(
  * @returns flattened JSON-LD AMF graph
  */
 async function generateGraph(ramlFilePath: string): Promise<object> {
-  ramlToolLogger.debug(`Generating flattened JSON-LD for ${ramlFilePath}`);
-  amf.plugins.document.WebApi.register();
-  amf.plugins.document.Vocabularies.register();
-  amf.plugins.features.AMFValidation.register();
-  await amf.Core.init();
+  const parsedRaml = await parseRamlFile(ramlFilePath);
 
-  //TODO: use common parser function that is used by validator and sdk
-  let model = await new amf.Raml10Parser().parseFileAsync(
-    `file://${ramlFilePath}`
-  );
-  model = new amf.Raml10Resolver().resolve(model);
+  const model = new amf.Raml10Resolver().resolve(parsedRaml);
```

Comment:
common/parser also has a resolveApiModel function you might want to use here

Reply:
I’ll take a look at this before merging.

Reply:
Given the todo around the pipelines in common/parser, I'm going to keep it like this for now. Once that todo gets resolved, this should be readdressed.